lib/add_magic_comment.rb

```diff
@@ -3,14 +3,15 @@
 # A simple library to prepend magic comments to multiple ".rb" files
 
 module AddMagicComment
-  MAGIC_COMMENT_PREFIX  = "frozen_string_literal"
-  MAGIC_COMMENT_PATTERN = /^-?# *#{MAGIC_COMMENT_PREFIX}/
+  MAGIC_COMMENT_PREFIX  = 'frozen_string_literal'
```

Comment:
Except you can't use `\` in single-quoted strings either. So now as a human you have to reason about whether a `\` appears in the string, as you did below for the 'erb' EXTENSION. It's not worth the thrash IMO.
(Now I remember disabling that Rubymine warning!)